test/unit_tests/modules/nav-primary-spec.js

```diff
@@ -45,61 +47,141 @@ describe( 'Get Event States', function() {
   } );
 
   describe( 'Primary Nav Link Events - Default state', function() {
+    beforeEach( function() {
+      $link = $( $( '.sub-nav-1' ).find( 'a' )[0] );
 
-    it( 'should not navigate away from the page', function() {
       $( '.nav-link-1' ).trigger( 'click' );
+    } );
 
+    it( 'should not navigate away from the page', function() {
       expect( window.location.href ).to.not.equal( 'http:// www.google.com/' );
```

Comment:
If the `event.preventDefault()` failed it would navigate to google.com because that's the url I used in the simplified nav on L:25